lib/src/providers/login_messages.dart

```diff
@@ -36,11 +39,18 @@ class LoginMessages with ChangeNotifier {
   static const defaultflushbarTitleError = 'Error';
   static const defaultSignUpSuccess = 'An activation link has been sent';
   static const defaultProvidersTitle = 'or login with';
+  static const defaultCompleteSignupButton = 'SUBMIT';
+  static const defaultCompleteSignupIntro =
+      'Please fill in this form to complete the signup';
 
   /// Hint text of the userHint [TextField]
   /// By default is Email
   final String userHint;
 
+  final String completeSignupButton;
+
+  final String completeSignupInfo;
```

Comment:
Can you please add a small comment about the type of hint

Reply:
done in da5d600

Reply:
I renamed them as well for consinstency with the ones of the recovery card